tests/go-validation/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -x
+set -eu
+
+DEBUG="${DEBUG:-false}"
+RKE_VERSION="${RKE_VERSION:-v1.0.2}"
+RANCHER_HELM_VERSION="${RANCHER_HELM_VERSION:-v3.6.0}"
+KUBECTL_VERSION="${KUBECTL_VERSION:-v1.16.6}"
+CLI_VERSION="${CLI_VERSION:-v2.4.5}"
+SONOBUOY_VERSION="${SONOBUOY_VERSION:-0.18.2}"
+TERRAFORM_VERSION="${TERRAFORM_VERSION:-0.12.10}"
```

Comment:
It looks like not all of these env vars are used, we may want to remove any unused ones. We should also verify that these are the minimum versions we should be targetting.

Reply:
I think we should just remove them for now, since I'm not using these en vars at all.

Reply:
removed them